mars/services/scheduling/api/oscar.py

```diff
@@ -47,7 +53,17 @@ async def create(cls: Type[APIType], session_id: str, address: str) -> APIType:
             SubtaskQueueingActor.gen_uid(session_id), address=address
         )
 
-        scheduling_api = SchedulingAPI(session_id, address, manager_ref, queueing_ref)
+        from ...cluster import ClusterAPI
+        from ..supervisor.autoscale import AutoscalerActor
+
+        cluster_api = await ClusterAPI.create(address)
+        supervisor_address = (await cluster_api.get_supervisors())[0]
+        autoscaler = await mo.actor_ref(
+            AutoscalerActor.default_uid(), address=supervisor_address
```

Comment:
Why not use `address=address`? The `address` is the current supervisor address.

Reply:
If supervisor created subpools and current process is subpool, then `address` won't be  supervisor address

Reply:
But, the `SubtaskQueueingActor`, `SubtaskManagerActor` and `AutoscalerActor` are all created in the same address. So, the actor refs should be created from the same address, too. Please refer to: https://github.com/mars-project/mars/blob/master/mars/services/scheduling/supervisor/service.py#L122

Can you unify the address of actor ref creations of `SubtaskQueueingActor`, `SubtaskManagerActor` and `AutoscalerActor`?

Reply:
`SubtaskQueueingActor` and `SubtaskManagerActor` may be created in the subpool. The `AutoscalerActor` will always be created in the supervisor main pool, it's similar to `GlobaleResourceManager`.

Reply:
> AutoscalerActor

It's different address, the address of`SubtaskQueueingActor`/`SubtaskManagerActor`/`AutoscalerActor` may be subpool address, `AutoscalerActor` will always use main pool address.

I changed actror_ref creation of `AutoscalerActor` to:
```python
cluster_api = await ClusterAPI.create(address)
[autoscaler] = await cluster_api.get_supervisor_refs(
    [AutoscalerActor.default_uid()]
)
```